index.d.ts

```diff
@@ -590,10 +590,15 @@ declare module 'react-native-health' {
   }
 
   export interface HealthActivityOptions
-    extends Omit<Omit<HealthValueOptions, 'unit'>, 'value'> {
+    extends Omit<HealthValueOptions, 'unit' | 'value'> {
     type: HealthActivity
   }
 
+  export interface HealthCategoryValueOptions {
```

Comment:
Didn't know if I should create a separate type of this and how to name. There is no [value](https://developer.apple.com/documentation/healthkit/hkcategoryvalue/hkcategoryvaluenotapplicable?language=objc) in `saveMindfulSession`